.github/workflows/build-golang-fedora.yaml

```diff
@@ -0,0 +1,34 @@
+# (C) Copyright Confidential Containers Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+name: "build golang-fedora container image"
+
+on:
+  workflow_dispatch:
```

Comment:
What if it additionally gets triggered on push (only and only if the dockerfile changed)? Relying on humans to push the button is always error prone....